src/screens/createWallet/CreateMnemonicScreen.tsx

```diff
@@ -41,66 +47,127 @@ function CreateMnemonicScreen(): JSX.Element {
   };
 
   return (
-    <>
-      <BackButton />
-      <Text fontSize="xl" mb={4}>
-        Create a new wallet
-      </Text>
-      <Button onClick={regenerateMnemonic} pt={2} pb={2} pl={4} pr={4}>
-        Generate new mnemonics
-      </Button>
-      <Card fontSize="sm" margin={[4, null]} borderRadius="xl" w="100%">
-        <CardHeader pb={0}>
-          <Text color="orange.500" textAlign="center">
+    <Flex
+      flexDir="column"
+      alignItems="center"
+      w={{ base: '100%', md: '75%' }}
+      maxW="4xl"
+    >
+      <Image src={logo} width={200} my={8} />
+
+      <Card
+        fontSize="sm"
+        marginY={4}
+        padding={0}
+        w={{ base: '100%', md: '90%' }}
+      >
+        <CardHeader pb={0} textAlign="center">
+          <Text
+            color="brand.green"
+            mb={4}
+            textAlign="center"
+            fontSize={{ base: '24px', md: '30px' }}
+            fontFamily="Univers63"
+          >
+            Create a new wallet
+          </Text>
+          <Text
+            as="b"
+            textAlign="center"
+            color="brand.lightGray"
+            fontSize={{ base: '16px', md: '20px' }}
+            fontFamily="Univers65"
+          >
             Please, save the mnemonic to the safe place and do not share with
             anyone.
           </Text>
         </CardHeader>
-        <CardBody>
-          <SimpleGrid columns={[2, null, 3]} spacing="10px">
-            {mnemonic.split(' ').map((word, idx) => (
-              <Box
-                // eslint-disable-next-line react/no-array-index-key
-                key={`${idx}_${word}`}
-                borderRadius="md"
-                borderWidth={1}
-                borderColor="whiteAlpha.800"
-                bg="whiteAlpha.50"
-                p={2}
-              >
-                <Text as="span" fontSize="xx-small">
-                  {idx + 1}.{' '}
-                </Text>
-                {word}
-              </Box>
-            ))}
+        <Text
+          as="u"
+          onClick={regenerateMnemonic}
+          paddingX={4}
+          paddingY={2}
+          textAlign="center"
+          cursor="pointer"
+        >
+          Generate new mnemonics
+        </Text>
+        <CardBody paddingY={10}>
+          <SimpleGrid columns={columns} spacing="0px">
+            {mnemonic.split(' ').map((word, idx) => {
+              const isLeftColumn = idx % columns === 0;
+              const isRightColumn = (idx + 1) % columns === 0;
+              const isTopRow = idx < columns;
+              const isBottomRow = idx >= mnemonic.split(' ').length - columns;
```

Comment:
So complicated stuff all around rendering lines...

Wouldn't grid + gap + bg color be enough?
<img width="327" alt="image" src="https://github.com/user-attachments/assets/5fc742c3-d27d-4352-83e0-33c53273eb06">
<img width="676" alt="image" src="https://github.com/user-attachments/assets/bb3fd9c1-d2ec-4d30-9212-9a564ae1c29f">